views/update/includes/entity.html

```diff
@@ -0,0 +1,156 @@
+{% set formattedPrincipalAddressHtml %}
+  {% set address = appData.entity.principal_address %}
+  {% include "includes/display_address.html" %}
+{% endset %}
+
+{% set formattedServiceAddressHtml %}
+  {% if appData.entity.is_service_address_same_as_principal_address == 1 %}
+    The correspondence address is the same as the entity's principal or registered office address
+  {% else %}
+    {% set address = appData.entity.service_address %}
+    {% include "includes/display_address.html" %}
+  {% endif %}
+{% endset %}
+
+{% set formattedPublicRegister %}
+  {% if appData.entity.is_on_register_in_country_formed_in == 1 %}
+    {{ appData.entity.public_register_name }} / {{ appData.entity.public_register_jurisdiction }} / {{ appData.entity.registration_number }}
+  {% else %}
+    No
+  {% endif %}
+{% endset %}
+
+<h2 class="govuk-heading-xl">Overseas entity details (NOT LIVE)</h2>
+
+{{ govukSummaryList({
+    rows: [
+      {
+        key: {
+          text: "Name of the overseas entity"
+        },
+        value: {
+          text: appData.entity.name
+        },
+        actions: {
+          items: [ CREATE_UPDATE_CHANGE_LINK(
+            OE_CONFIGS.OVERSEAS_ENTITY_UPDATE_DETAILS_URL,
```

Comment:
We need to pass in a variable instead of using a constant here ( OE_CONFIGS.OVERSEAS_ENTITY_UPDATE_DETAILS_URL). Then we can potentially use this view for both registration and update.